app/controllers/admin_controller.rb

```diff
@@ -371,6 +371,11 @@ def queue
     end
   end
 
+  def test_digest_email_spam
+    DigestSpamJob.perform_async(0)
```

Comment:
What does `perform_async` do here? Does it send the email in the background?

Reply:
Yup, It is just for testing. It performs Job and creates Digest on daily basis. perform_async gets pushed to the Queue right away. 0 is for daily digest and 1 is for a weekly digest